USF-1975: Synchronize Checkout_PlaceOrder documentation (#192)
commerce-docs/microsite-commerce-storefront: src/content/docs/get-started/release.mdx

  - The `icon` configuration specifies the name of the icon to be shown beside of the label. The icon name must exist within the list of available icons defined in the drop-ins SDK.
  - The `setOnChange` configuration sets the payment method automatically when it is selected. Only if a payment method is specifically set to _false_, the container will not automatically set the payment method to the cart when selected (for example, if a payment method needs to obtain more information during the place order action).
  - The `render` configuration is a handler used to render and configure the payment method.

#### v1.2.0

_February 11, 2025_

> The `PlaceOrder` container configuration structure has been modified. It has been added a new property into the `PlaceOrderProps` interface:

Reviewer: ```suggestion
The `PlaceOrder` container configuration structure has been modified. A new property is added to the `PlaceOrderProps` interface:
```
